path/src/cmd/background.ps1

```
@@ -0,0 +1,22 @@
+﻿function script:cmd_background {
+	require passthrough win/wt
+	$env:FOUNT_BACKGROUND = 1
+	handle_docker_passthrough @args
+	$args = @($args | Select-Object -Skip 1)
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'args' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)